### PR TITLE
Nerfs Blueshits, reminds shitters to not be shitters

### LINKED ISSUE #529: Blueshield ID has ID modification access
With a blueshield ID you can enter the HOP office, which it also has access to, unlock their locker, put in a new ID, and give it all access and yourself, all access from just your ID.

### PR DESCRIPTION
**Pull request by ike709**
Fixes #529 

:cl: ike709
add: Some roles now have reminders of what not to do.
balance: Blueshields are now timelocked at ten hours as crew.
balance: Blueshields can no longer use identification consoles to alter IDs.
/:cl:

A role that often wanders the station as an all access rambo is no bueno.

Also, no clue what the hell is wrong with blueshield.dm formatting, but good luck playing spot the difference.

Example of a special notice (EDIT: Different words in the latest version):
image

### CODE REVIEW

**Comment by ike709** [comment]
Now taking suggestions for any other roles that need special notices.

File: code/modules/jobs/job_types/civilian.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@ Clown
 	spawn_positions = 1
 	supervisors = "the head of personnel"
 	selection_color = "#dddddd"
+	special_notice = "There is a difference between harmless pranks and griefing. Know it."
```
Comment (Ziiro) [inline code/modules/jobs/job_types/civilian.dm]: 👍

File: code/modules/jobs/job_types/blueshield.dm [hunk 2]
```diff
@@ -3,27 +3,30 @@ Blueshield
...
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "captain and command personnel"
+	selection_color = "#ddddff"
+	req_admin_notify = 1
+	minimal_player_age = 14
+	exp_requirements = 600 //10 hours
+	exp_type = EXP_TYPE_CREW
+	special_notice = "You are not a security officer. Do not do their job."
```
Comment (Xantholne) [inline code/modules/jobs/job_types/blueshield.dm]: Maybe change to Reminder, you are a Bodyguard to Heads of Staff. Not a Security Officer. Do not do their job.

Comment (ike709) [reply code/modules/jobs/job_types/blueshield.dm]: Done.

**Comment by FantasticFwoosh** [comment]
Wouldn't asking for explicit supervisory guidance (from command) before doing anything further into security officer work be appropriate to cover this psuedo policy issue?

The worst case scenarios is that blueshields will take this as a cue to be ambivalent in a actual emergency because of a "Its not my job" or "I dont wan't to get banned, you do it" attitude, sort of how the detective may require to step up in emergencies under circumstances and pull weight with the rest of the security team.

File: code/modules/jobs/job_types/security.dm [hunk 3]
```diff
@@ -126,6 +126,7 @@ Detective
 	minimal_player_age = 7
 	exp_requirements = 300
 	exp_type = EXP_TYPE_CREW
+	special_notice = "You are not a security officer. Do not do their job."
```
Comment (FlattestGuitar) [inline code/modules/jobs/job_types/security.dm]: The Detective kinda is expected to help the Officers in most situations. I think it's a bit silly to give them a baton, cuffs and a gun and then tell them not to help the people he works with constantly.

File: code/modules/jobs/job_types/civilian.dm (hunk 1 above)
Comment (FlattestGuitar) [reply code/modules/jobs/job_types/civilian.dm]: I'd add this to the mime as well.

File: code/modules/jobs/job_types/security.dm (hunk 3 above)
Comment (Ziiro) [reply code/modules/jobs/job_types/security.dm]: `You are not a security officer, do not do their job for them. You should assist them if they need help, however.`

Comment (ike709) [reply code/modules/jobs/job_types/security.dm]: New one is slightly different from Ziiro's proposed change.

**Pull request merged**